etc/workflows/duplicate-event.xml

```diff
@@ -85,6 +85,21 @@
       </script>
     ]]>
   </configuration_panel>
+  <configuration_panel_json>
```

Comment:
There are a bunch of fields which show up in the old admin ui, but not the new one for this workflow (which only shows the number of dups, as defined here).  This causes things to be unhappy, but work:
```
2023-04-20 15:23:01,481 | INFO  | (DuplicateEventWorkflowOperationHandler:281) - Creating 4 new media packages from media package with id 0e583a0f-d1bc-450d-8f85-23aa5d95d176.
2023-04-20 15:23:01,487 | WARN  | (DuplicateEventWorkflowOperationHandler:371) - Could not parse: ${startDateTime} as date time
2023-04-20 15:23:01,488 | WARN  | (DuplicateEventWorkflowOperationHandler:373) - Using original event date Thu Apr 20 14:58:00 CST 2023 as default
```

Reply:
It seems the new Admin UI currently only supports workflow config fields of types `radio`, `checkbox`, `number` and `text`. So I can't just add field of type `datetime-local` to the `configuration_panel_json`, as it is unsupported (I mean, I tried anyway, but of course it didn't work).

I'll open an issue for that. Do you want this fixed before merging?

Reply:
Nope, this is a beta so the occasional issue is to be expected.  This would block promotion of Picard to the default ui though.